app/routes/photos/components/GalleryEditor.js

```diff
@@ -256,19 +257,20 @@ class GalleryEditor extends Component<Props, State> {
             justifyContent="flex-end"
           >
             {!isNew && (
-              <Button
-                onClick={this.onDeleteGallery}
-                className={styles.deleteButton}
+              <ConfirmModalWithParent
+                title="Slett album"
+                message="Dette vil slette hele albumet og alle bildene inni"
+                onConfirm={this.onDeleteGallery}
               >
-                Delete
-              </Button>
+                <Button className={styles.deleteButton}>Slett album</Button>
+              </ConfirmModalWithParent>
             )}
             <Button
               disabled={submitting}
               className={styles.submitButton}
               type="submit"
             >
-              {isNew ? 'Create' : 'Save'}
+              {isNew ? 'Lag nytt' : 'Lagre'}
```

Comment:
👍

Reply:
Maybe "Opprett" instead of "Lag nytt"? Subjective, of course

Reply:
Agreed